docs/snapshotting/snapshot-support.md

```diff
@@ -38,7 +42,17 @@ This has the advantage of very fast snapshot loading times, but comes with the c
 of having to keep the guest memory file around for the entire lifetime of the
 resumed microVM.
 
-*Note*: Snapshotting is currently supported only on `x86_64` machines.
+## Performance
+
+The Firecracker snapshot create/resume performance depends on the memory size,
+vCPU count and emulated devices count. The Firecracker CI runs snapshots tests on AWS **m5d.metal** instances and the baseline for snapshot resume latency target is under **8ms** with 5ms p90 for a microvm with this specs: 2vCPU/512MB/1 block/1 net device.
+
+## Known issues and limitations
+
+- High snapshot latency on 5.4+ host kernels - [#2129](https://github.com/firecracker-microvm/firecracker/issues/2129)
+- Guest network connectivity is not guaranteed to be preserved after resume
+- Vsock connectivity is broken after resume
+- Poor entropy and replayable randomness when resuming multiple microvms which deal with cryptographic secrets
```

Comment:
That's a feature not a bug 😁 .

Joke aside, I think there's no way around that, and before general availability we should put out write proper documentation/examples on how to reseed randomness, e.g., after doing more than one restore from a microVM.  You can also link to the snapshot safety section in the corresponding doc.

Reply:
Linked the content from PR #2138, but it is broken until it gets merged.